nixos/modules/services/web-apps/netbox.nix

```diff
@@ -0,0 +1,249 @@
+{ lib
+, config
+, pkgs
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.netbox;
+  package = pkgs.netbox.override {
+    configFile = configFile;
+  };
+
+  gunicornConfigFile = pkgs.writeText "netbox-gunicorn.py" ''
+    # The IP address (typically localhost) and port that the Netbox WSGI process should listen on
+    bind = 'unix:/run/netbox/netbox.sock'
+
+    # Number of gunicorn workers to spawn. This should typically be 2n+1, where
+    # n is the number of CPU cores present.
+    workers = 5
+
+    # Number of threads per worker process
+    threads = 3
+
+    # Timeout (in seconds) for a request to complete
+    timeout = 120
+
+    # The maximum number of requests a worker can handle before being respawned
+    max_requests = 5000
+    max_requests_jitter = 500
+  '';
```

Comment:
My approached apparently used a gunicorn config file which is in upstreams contrib directory.

https://github.com/netbox-community/netbox/blob/develop/contrib/gunicorn.py